src/common/ImageAvatar.js

```diff
@@ -15,14 +15,15 @@ const styles = StyleSheet.create({
 });
 
 export default({ avatarUrl, size, status, isCircular, onPress }) => {
-  const imageStyle = {
+  const touchableStyle = {
     height: size,
     width: size,
-    borderRadius: isCircular ? size / 2 : size / 8,
   };
 
+  const imageStyle = { ...touchableStyle, borderRadius: isCircular ? size / 2 : size / 8};
```

Comment:
Good to know! Thanks.

Checked it out - imageStyle object does not have to inherit.